include/vicon_transformer/pointcloud.hpp

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/**
+ * @file
+ * @brief Utility functions for computing transform between point clouds.
+ * @copyright 2023 Max Planck Gesellschaft.  All rights reserved.
+ */
+#pragma once
+
+#include <Eigen/Eigen>
+
+#include <vicon_transformer/thirdparty/json.hpp>
```

Comment:
I'm always a bit unclear about what is the best convention with this.  To my understanding, the only technical difference is that with quotes it will first try to interpret it as relative include and if that fails will fall back to same behaviour as <> (i.e. searching the include paths).  Therefore I typically only use quotes for actual relative includes and <> for all else.
I don't have a strong opinion on this, though.

Reply:
I did some more research on this but did not really come to a conclusion.
The [C++ Core Guidelines Rule SF.12](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#sf12-prefer-the-quoted-form-of-include-for-files-relative-to-the-including-file-and-the-angle-bracket-form-everywhere-else) covers this and seems to somewhat support my current convention:
> ...the guidance is to use the quoted form for including files that exist at a relative path to the file containing the #include statement (from within the same component or project) and to use the angle bracket form everywhere else, where possible.

Apart from this, there does not seem to be a general consensus in the C++ community.

Reply:
but isnt it a relative path ?

Reply:
Oooh, I missed that this is in a header file.  As is, it's not really relative (due to the `vicon_transformer/` part), but it could (and probably should) be made relative in this case.  I'll change it.

Only in case of cpp files that are not in the same directory as the headers (i.e. the relative path would be something like `../include/vicon_transformer/...`), I would stick to <> includes.